capstone/src/components/User/Experience/RegisterExperience/ClaimRestaurant.jsx

```diff
@@ -0,0 +1,99 @@
+import {
+  Heading,
+  Text,
+  Image,
+  Button,
+  HStack,
+  VStack,
+  Badge,
+} from '@chakra-ui/react';
+
+const API_KEY = process.env.REACT_APP_GOOGLE_MAPS_API_KEY;
+
+export default function ClaimRestaurant({
+  restaurants,
+  experienceValues,
+  onUpdate,
+  claimStatus,
+  onSelect,
+}) {
+  /* If there are not restaurants to show, or if the user has claimed but there is
+  not a type (since he has not made any modification on his actual data) don't display anything */
+  if (
+    restaurants.length === 0 ||
+    (claimStatus.hasClaimed && claimStatus.type === '')
+  ) {
+    return null;
+  }
+  // the user has modified his data and also claimed a status
+  if (claimStatus.hasClaimed) {
+    return (
+      <Badge colorScheme="purple">
+        {claimStatus.type} experience registered
+      </Badge>
+    );
+  }
+  // Disabled to match google maps format
+  // eslint-disable-next-line camelcase
+  function onClaim({ address, lat, lng, place_id }) {
```

Comment:
You can rename destructured variables, that might help bypass the lint rule.

Alternatively you could disable the rule at the module level.

```suggestion
  function onClaim({ address, lat, lng, place_id: placeID }) {
```

Reply:
oh, great! I did it for the module level!